src/components/SatelliteManifestPanel/SatelliteManifestPanel.tsx

```diff
@@ -275,10 +275,6 @@ const SatelliteManifestPanel: FunctionComponent<SatelliteManifestPanelProps> = (
           <Drawer isExpanded={detailsDrawerIsExpanded} className="sub-c-drawer-satellite-manifest">
             <DrawerContent panelContent={panelContent()}>
               <DrawerContentBody>
-                <Title headingLevel="h2">
```

Comment:
We don't seem to be using Title or Badge anywhere else on the page, so you should be able to remove those from the import statements.

Also, could you please remove the changes to package-lock.json from this pr?  I don't get any diff when I npm install from main, so you may be using the wrong version of something.  I'm using node version v14.20.0 and npm version 6.14.17.  Be sure you are using approximately those versions.

Reply:
I have updated the imports, and updated the npm package-lock.json files issue.